packages/ocplib-json-typed/ocplib-json-typed.0.7.1+dune/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+synopsis: "Libraries for reliable manipulation JSON objects"
+maintainer: "Benjamin Canou <benjamin@ocamlpro.com>"
+authors: "Benjamin Canou <benjamin@ocamlpro.com>"
+homepage: "https://github.com/dune-universe/ocplib-json-typed"
+bug-reports: "https://github.com/ocamlpro/ocplib-json-typed/issues"
+license: "LGPLv3 w/ linking exception"
+dev-repo: "git+https://github.com/dune-universe/ocplib-json-typed.git"
+
+build: [
+  ["dune" "build" "-j" jobs "-p" name "@install"]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {build & >= "1.7"}
```

Comment:
I vaguely remember that opam-repository maintainers asked for the build flag to be removed from the dependency on `dune`.

Reply:
Ah yes. The upstream tarball has it (since it ships with dune files), but we can certainly patch it out here :+1: